test/extensions/filters/udp/dns_filter/dns_filter_test.cc

```diff
@@ -1845,7 +1845,8 @@ TEST_F(DnsFilterTest, InvalidShortBufferTest) {
   EXPECT_EQ(1, config_->stats().downstream_rx_invalid_queries_.value());
 }
 
-TEST_F(DnsFilterTest, RandomizeFirstAnswerTest) {
+// FIXME (Maistra): See https://github.com/envoyproxy/envoy/pull/24330
+TEST_F(DnsFilterTest, DISABLED_RandomizeFirstAnswerTest) {
```

Comment:
Do you want to track this in our Jira? If so, create an issue and mention it here.

BTW, next time it might be easier to just cherry-pick the commit from one branch to another, e.g. `git cherry-pick d032ce917328e388d40c3fe5ba1a49c3b570486d`. This preserves the original metadata (title, author, etc).